dataflow/src/main/java/org/checkerframework/dataflow/livevariable/LiveVariableTransfer.java

```diff
@@ -0,0 +1,100 @@
+package org.checkerframework.dataflow.livevariable;
+
+import java.util.List;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.BackwardTransferFunction;
+import org.checkerframework.dataflow.analysis.RegularTransferResult;
+import org.checkerframework.dataflow.analysis.TransferInput;
+import org.checkerframework.dataflow.analysis.TransferResult;
+import org.checkerframework.dataflow.cfg.UnderlyingAST;
+import org.checkerframework.dataflow.cfg.node.AbstractNodeVisitor;
+import org.checkerframework.dataflow.cfg.node.AssignmentNode;
+import org.checkerframework.dataflow.cfg.node.MethodInvocationNode;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.dataflow.cfg.node.ObjectCreationNode;
+import org.checkerframework.dataflow.cfg.node.ReturnNode;
+import org.checkerframework.dataflow.cfg.node.StringConcatenateAssignmentNode;
+
+/**
+ * An implementation of a transfer function using for live variable analysis. To run live variable
+ * analysis, see {@link org.checkerframework.dataflow.cfg.playground.LiveVariablePlayground}.
+ */
+public class LiveVariableTransfer
```

Comment:
For consistency, shouldn't this be `LiveVarTransfer`? Or use `Variable` also in the other classes?